### DISCUSSION TITLE
bypy uploading velocity

### DISCUSSION THREAD

**Discussion by LouisSu123** [houtianze/bypy#628] [Q&A]
bypy上传文件的速度好慢（只有不到0.5M每秒），我是开了会员的，请问有什么办法可以设置一下吗？谢谢大家！

**Comment by liangwx** [comment]
试试结合aria2，看能速度达到多少

**Comment by caite425** [comment]
确实，到目前为止上传速度是唯一的硬伤，由于ARM的ubuntu无法安装客户端，这也是使用的唯一方法了，秒传功能其实也是很好用的。作者辛苦了！

**Reply by houtianze** [reply]
最新的版本修复了多进程，可以更新到最新版，试一试加 --progresses 参数。这个只对多个文件/文件夹有用，单个文件无法提速。

**Reply by LouisSu123** [reply]
是的是的，这个确实还是很好用的
